src/ucp/api/ucp.h

```diff
@@ -2100,6 +2123,33 @@ ucs_status_t ucp_context_query(ucp_context_h context_p,
                                ucp_context_attr_t *attr);
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief Compare two remote keys
+ *
+ * This routine returns equal if all the transports can compare equal and
```

Comment:
` This routine returns equal if all the transports can compare equal and configurations are equal.` - this is very puzzling.  What are you trying to state here ?
`This routine compares two remote keys`.

Reply:
replaced sentence with `This routine compares two remote keys`.